index.html

```diff
@@ -326,13 +269,127 @@ <h3>
+       "ethereumAddress": "0xb9c5714089478a327f09197987f16f9e5d936e8a"
+    }],
+    "authentication": [
+       "did:web:example.com#controller"
+    ]
+  }
+}
+          </pre>
+  
+        <pre class="example" title="Example did:web DID Resolution Document using JWS2020">
+{
+  "@context": "https://w3id.org/did-resolution/v1",
+  "didDocument": {  
+    "@context": [
+      "https://www.w3.org/ns/did/v1",
+      "https://w3id.org/security/suites/jws-2020/v1"
+    ],
+    "id": "did:web:example.com",
+    "verificationMethod": [
+      {
+        "id": "did:web:example.com#key-0",
+        "type": "JsonWebKey2020",
+        "controller": "did:web:example.com",
+        "publicKeyJwk": {
+          "kty": "OKP",
+          "crv": "Ed25519",
+          "x": "0-e2i2_Ua1S5HbTYnVB0lj2Z2ytXu2-tYmDFf8f5NjU"
+        }
+      },
+      {
+        "id": "did:web:example.com#key-1",
+        "type": "JsonWebKey2020",
+        "controller": "did:web:example.com",
+        "publicKeyJwk": {
+          "kty": "OKP",
+          "crv": "X25519",
+          "x": "9GXjPGGvmRq9F6Ng5dQQ_s31mfhxrcNZxRGONrmH30k"
+        }
+      },
+      {
+        "id": "did:web:example.com#key-2",
+        "type": "JsonWebKey2020",
+        "controller": "did:web:example.com",
+        "publicKeyJwk": {
+          "kty": "EC",
+          "crv": "P-256",
+          "x": "38M1FDts7Oea7urmseiugGW7tWc3mLpJh6rKe7xINZ8",
+          "y": "nDQW6XZ7b_u2Sy9slofYLlG03sOEoug3I0aAPQ0exs4"
+        }
+      },
+    ],
+    "authentication": [
+      "did:web:example.com#key-0",
+      "did:web:example.com#key-2"
+    ],
+    "assertionMethod": [
+      "did:web:example.com#key-0",
+      "did:web:example.com#key-2"
+    ],
+    "keyAgreement": [
+      "did:web:example.com#key-1", 
+      "did:web:example.com#key-2"
+    ]
+  }
+}
+          </pre>
+  
+          <pre class="example" title="Example did:web JSON-LD DID Resolution Document with a cryptographic signature">
+{
+  "@context": "https://w3id.org/did-resolution/v1",
+  "didDocument": {
+    "@context": "https://www.w3.org/ns/did/v1",
+    "id": "did:web:example:com",
+    "authentication": [{
+      "id": "did:web:example:com#keys-1",
+      "type": "Ed25519VerificationKey2018",
+      "controller": "did:web:example:com",
+      "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+    }],
+    "service": [{
+      "id": "did:web:example:com#vcs",
+      "type": "VerifiableCredentialService",
+      "serviceEndpoint": "https://example.com/vc/"
+    }],
+  },
+  "didDocumentMetadata": {
+    "created": "2019-03-23T06:35:22Z",
+    "updated": "2023-08-10T13:40:06Z",
+    "proof": {
+      "type": "Ed25519Signature2020",
+      "created": "2022-08-21T12:54:33Z",
+      "verificationMethod": "did:web:example.com:trustedSigner#WEB",
+      "proofPurpose": "assertionMethod",
+      "proofValue": "z3e7VcjAVz7RVb7uuH7NQzDuF9AXK3vfH65xnSJSRiLK3vTPUoCDgcE4JC1UfZhbD1wgFsWxQwdWEtrRqhbnqvyYs"
+    }
```

Comment:
-1 to detaching the `proof` in this way. You can include it in `didDocument`, you just need to add the appropriate context file. If we detach the `proof` in this way, we'll have to have special processing instructions just for did:web resolution which would be special just for did:web.

Please move `proof` to the `didDocument` and add the appropriate `@context` definition to make it clean.